test/test_transforms_v2_refactored.py

```diff
@@ -351,45 +351,70 @@ def assert_warns_antialias_default_value():
         yield
 
 
-def reference_affine_bounding_boxes_helper(bounding_boxes, *, format, canvas_size, affine_matrix):
-    def transform(bbox):
+def reference_affine_bounding_boxes_helper(bounding_boxes, *, affine_matrix, expand=False):
+    def affine_bounding_boxes(bounding_boxes):
```

Comment:
Nit: here method name is unclear what it does as `affine_bounding_boxes` reads as a noun and not a verb

Reply:
This is the same name as our kernel:

https://github.com/pytorch/vision/blob/fae532174e1cde24542d3b357055d8f37b44e326/torchvision/transforms/v2/functional/_geometry.py#L820